### PR TITLE
New plugin

### PR DESCRIPTION
**Pull request by lechu2375**

### CODE REVIEW

File: sh_legshoot.lua [hunk 1]
```diff
@@ -0,0 +1,28 @@
+PLUGIN.name = "Legs wyjebnik"
```
Comment (impulsh) [inline sh_legshoot.lua:1]: Full English title would be preferred

File: sh_legshoot.lua [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+PLUGIN.author = "Lechu2375"
+PLUGIN.description = "Now hitting right in the legs causes injures."
+PLUGIN.license =  "MIT not for use on Kaktusownia"
```
Comment (impulsh) [inline sh_legshoot.lua:4]: The full MIT license text is required

File: sh_legshoot.lua [hunk 3]
```diff
@@ -0,0 +1,28 @@
+PLUGIN.name = "Legs wyjebnik"
+PLUGIN.author = "Lechu2375"
+PLUGIN.description = "Now hitting right in the legs causes injures."
```
Comment (impulsh) [inline sh_legshoot.lua:3]: You should describe how to customize the plugin (e.g how to add ignored factions) in `PLUGIN.readme`

File: sh_legshoot.lua [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+PLUGIN.license =  "MIT not for use on Kaktusownia"
+if SERVER then
+    local legs = {
+        [HITGROUP_LEFTLEG] = true,
+        [HITGROUP_RIGHTLEG] = true
+    }
+    local resFactions = {  --table with factions enums that should be resistant to these effects
+        [FACTION_OTA] = true
```
Comment (impulsh) [inline sh_legshoot.lua:11]: This should probably be made a comment since you can't be sure that users will have `FACTION_OTA` in their schema

**Review by impulsh — CHANGES_REQUESTED** [review]

**Comment by lechu2375** [comment]
https://github.com/lechu2375/helix-lechu2375

**Pull request closed without merging**